src/base/parseGPR.m

```diff
@@ -8,51 +8,61 @@
 % 4. 'and' and 'or' operators as well as gene names have to be followed and preceded by either a
 % whitespace character or a opening or closing bracket, respectively. Gene
 % Names can also be at the beginning or the end of the string.
-% 
-% 
+%
+%
 % USAGE:
 %
-%    [newGeneList,totalGeneList,ruleString] = generateRules(grRuleString,currentGenes)
+%    [newGeneList,totalGeneList,ruleString] = generateRules(grRuleString,totalGeneList)
 %
 % INPUT:
 %    grRuleString:     The rule string in textual format.
```

Comment:
Also this is no longer true. If its a cell array now, it expects a different input, maybe this should be marked with an additional flag?

Reply:
What exactly is no longer true?

Reply:
Essentially I'm wondering about line 40:
`if ~iscell..`
Why is this check necessary?

Reply:
This check is necessary because of lines 20 and 31 of `generateRules`.